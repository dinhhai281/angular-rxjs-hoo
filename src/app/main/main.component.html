<mat-toolbar color="primary">
  <a routerLink="/" class="mr-20">Angular - RxJS Part 3</a>
</mat-toolbar>
<router-outlet></router-outlet>
