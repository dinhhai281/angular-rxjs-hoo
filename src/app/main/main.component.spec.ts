import { NO_ERRORS_SCHEMA } from '@angular/core';
import { RouterTestingModule } from '@angular/router/testing';
import { render, RenderResult } from '@testing-library/angular';

import { MainComponent } from './main.component';

describe('MainComponent', () => {
  let component: RenderResult<MainComponent>;

  beforeEach(async () => {
    component = await render(MainComponent, {
      imports: [RouterTestingModule],
      schemas: [NO_ERRORS_SCHEMA],
    });
  });

  it('should match snapshot', () => {
    expect(component.container).toMatchSnapshot();
  });
});
