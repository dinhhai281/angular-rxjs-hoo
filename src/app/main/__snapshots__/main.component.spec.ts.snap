// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainComponent should match snapshot 1`] = `
<div
  id="root0"
>
  <mat-toolbar
    color="primary"
  >
    <a
      class="mr-20"
      href="/"
      routerlink="/"
    >
      Angular - RxJS Part 3
    </a>
  </mat-toolbar>
  <router-outlet />
</div>
`;
